README.md

```diff
@@ -180,6 +180,32 @@ windows:
 '------------------'------------------'
 ```
 
+### Two pane window with focus in second pane
+
+```yaml
+windows:
+  - name: sample-two-panes
+    root: ~/Code/sample/www
+    layout: even-horizontal
+    panes:
+      - rails server
+      - { commands: [ 'rails console' ], focus: true }
```

Comment:
Ah yes, of course.  Gosh, this is humiliating; I seldom use YAML, one might expect me to think about it before PR'ing on a YAML project. :-)  Thanks for your patience.  Updated.